packages/webapp/src/components/Table/TableV2.jsx

```diff
@@ -92,6 +93,11 @@ export default function TableV2(props) {
     shouldFixTableLayout,
     defaultOrderBy,
     alternatingRowColor,
+    onCheck,
+    handleSelectAllClick,
```

Comment:
I wonder if these two could be combined and simplified by passing a `setSelectedIds` function prop instead, and letting the table handle the different scenarios where the selected IDs need to change

Reply:
It's possible, but I'm hesitant to do it.
A few of my concerns:
- flexibility - what if we want to do something else when checkbox is clicked?
- duplicated `handleSelectAllClick` - we will have a function for this in its parent component anyways because we have "clear selection" in the action menu

Are they not a big deal??